airflow/providers/google/cloud/transfers/bigquery_to_gcs.py

```diff
@@ -132,6 +135,8 @@ def __init__(
         self.force_rerun = force_rerun
         self.reattach_states: set[str] = reattach_states or set()
         self.hook: BigQueryHook | None = None
+        self.deferrable = deferrable
+        self.configuration: dict = {}
```

Comment:
Why is this needed? From what I can tell you can simply pass `configuration` as an argument to `_submit_job`.